src/__tests__/__views__/EditProfile.test.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { mount } from "enzyme";
```

Comment:
For unit testing, ```shallow``` is the best. But I will have to research more about ```shallow``` and ```mount``` rendering in enzyme.

Reply:
So far so good!!!